index.js

```diff
@@ -1,13 +1,46 @@
 /* jshint node: true */
 'use strict';
 
+var writeFile = require('broccoli-file-creator');
+var mergeTrees = require('broccoli-merge-trees');
+
 var defaultBeacon = 'bam.nr-data.net';
 var defaultAgentClassic = 'js-agent.newrelic.com/nr-892.min.js';
 var defaultAgentSPA = 'js-agent.newrelic.com/nr-spa-963.min.js';
 
 module.exports = {
   name: 'ember-new-relic',
 
+  outputPath: 'new-relic.js',
+
+  newRelicConfig: null,
+
+  importToVendor: true,
+
+  included: function(app) {
+    this._super.included ? this._super.included.apply(this, arguments) : this._super(app);
+
+    var env  = process.env.EMBER_ENV;
+    var options = (this.app && this.app.options && this.app.options['ember-new-relic']) || {};
```

Comment:
It will return the last truthy value, so it will return the value of `this.app.options['ember-new-relic']`.

An example of this would be:

```javascript
var options = {};

console.log((options && options.foo) || 'nope');

options.foo = {baz: true};
console.log((options && options.foo) || 'nope');
```